lib/bokken_web/controllers/availability_controller.ex

```diff
@@ -0,0 +1,63 @@
+defmodule BokkenWeb.AvailabilityController do
+  use BokkenWeb, :controller
+
+  alias Bokken.Events
+  alias Bokken.Events.Availability
+
+  action_fallback BokkenWeb.FallbackController
+
+  defguard is_mentor(conn) when conn.assigns.current_user.role === :mentor
+
+  def show(conn, %{"id" => availability_id}) do
+    availability = Events.get_availability(availability_id)
```

Comment:
```suggestion
    availability = Events.get_availability!(availability_id)
```

This way you can remove the if statement, because when it fails, it will already give the correct error.